src/js/profile/wearable/widget/wearable/Selector.js

```diff
@@ -334,7 +334,7 @@
 					ITEM_RADIUS: -1,
 					ITEM_START_DEGREE: 30,
 					ITEM_END_DEGREE: 330,
-					ITEM_NORMAL_SCALE: "scale(" + STATIC.SCALE_FACTOR + ")",
+					ITEM_NORMAL_SCALE: "scale(" + STATIC.SCALE_FACTOR.toString() + ")",
```

Comment:
I don't think this file should be included

Reply:
Thanks. This is change from another commit. Will be remove.